Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM alpine
+FROM alpine:3.10
```

Comment:
What do you think about using 3.11.6? I think that's the last version of alpine prior to 3.12 at the moment.

Reply:
Yeah. I can switch to 3.11.6. It is kinda larger though.

Reply:
Done